test.disabled/ejabberd_tests/tests/mod_global_distrib_SUITE.erl

```diff
@@ -242,6 +260,12 @@ generic_end_per_testcase(CaseName, Config) ->
 %% Service discovery test
 %%--------------------------------------------------------------------
 
+test_advertised_endpoints_override_endpoints(Config) ->
+    GetEndpoints = fun({NodeName, _, _}) ->
+                            rpc(europe_node1, mod_global_distrib_mapping_redis, get_endpoints, [<<"fed1">>]) end,
```

Comment:
I think we want to call `NodeName`, not `europe_node1` every time? :)

Reply:
Sure :p